DREAM3DReviewFilters/FindArrayStatistics.cpp

```diff
@@ -257,6 +274,17 @@ void FindArrayStatistics::dataCheck()
 
   EXECUTE_FUNCTION_TEMPLATE(this, createCompatibleArrays, m_InputArrayPtr.lock())
 
+  if(m_FindHistogram)
+  {
+    std::vector<size_t> cDims_List(1, m_NumBins);
```

Comment:
```suggestion
    std::vector<size_t> cDims_List = {static_cast<size_t>(m_NumBins)};
```
Using an initializer list here is a bit more clear.